nf_core/pipeline-template/{{cookiecutter.name_noslash}}/main.nf

```diff
@@ -61,46 +54,46 @@ check_awsbatch()               // Check AWS batch settings
 summary = create_summary()     // Print parameter summary
 
 /*
- * FastQC
+ * Include local pipeline modules
  */
-//include 'modules/nf-core/fastqc' params(params)
+include { OUTPUT_DOCUMENTATION } from './modules/local/output_documentation' params(params)
+include { GET_SOFTWARE_VERSIONS } from './modules/local/get_software_versions' params(params)
+include { CHECK_SAMPLESHEET; check_samplesheet_paths } from './modules/local/check_samplesheet' params(params)
 
 /*
- * MultQC
+ * Include nf-core modules
  */
-//include 'modules/nf-core/multiqc' params(params)
+include { FASTQC } from './modules/nf-core/fastqc' params(params)
+include { MULTIQC } from './modules/nf-core/multiqc' params(params)
 
 /*
- * Get software versions
+ * Run the workflow
  */
-include 'modules/nf-core/get_software_versions' params(params)
-get_software_versions()
+workflow {
 
-/*
- * Output markdown documentation as HTML
- */
-include 'modules/nf-core/output_documentation' params(params)
-output_documentation()
+    CHECK_SAMPLESHEET(ch_input)
+        .splitCsv(header:true, sep:',')
+        .map { check_samplesheet_paths(it) }
+        .set { ch_raw_reads }
+
+    FASTQC(ch_raw_reads)
+
+    OUTPUT_DOCUMENTATION(
+        ch_output_docs,
+        ch_output_docs_images)
+
+    GET_SOFTWARE_VERSIONS()
+
+    // MULTIQC(
+    //     summary,
+    //     fastqc.out,
+    //     ch_multiqc_config
+    // )
```

Comment:
How come this is commented out?

Reply:
It hasn't been implemented properly yet because it needs various variables from the pipeline e.g. `summary`, `custom_runName` etc and so needs a little more work. May be easier to figure this out after we have dealt with the Groovy code because that will determine how these variables are passed around?